roc-toolkit fails to build on aarch64-linux

### Describe the bug

roc-toolkit fails to build on aarch64-linux, with the following error:

```
In file included from /nix/store/w65pqng7l4f706bk9j4m7y6yrvdqs6kd-libunwind-1.6.2-dev/include/libunwind.h:7,
                 from src/modules/roc_core/target_libunwind/roc_core/backtrace.cpp:10:
/nix/store/w65pqng7l4f706bk9j4m7y6yrvdqs6kd-libunwind-1.6.2-dev/include/libunwind-aarch64.h:198:10: error: expected identifier before numeric constant
  198 |  alignas(16) uint8_t __reserved[(66 * 8)];
      |          ^~
/nix/store/w65pqng7l4f706bk9j4m7y6yrvdqs6kd-libunwind-1.6.2-dev/include/libunwind-aarch64.h:198:10: error: expected ',' or '...' before numeric constant
/nix/store/w65pqng7l4f706bk9j4m7y6yrvdqs6kd-libunwind-1.6.2-dev/include/libunwind-aarch64.h:198:12: error: expected ';' at end of member declaration
  198 |  alignas(16) uint8_t __reserved[(66 * 8)];
      |            ^
      |             ;
scons: *** [build/aarch64-pc-linux-gnu/gcc-9.3.0-release/modules/roc_core/target_libunwind/roc_core/backtrace.o] Error 1
scons: building terminated because of errors.
```

### Steps To Reproduce

```
nix-env -f channel:nixpkgs-unstable -iA roc-toolkit
```

### Expected behavior
roc-toolkit should build successfully.

### Screenshots

N/A

### Additional context

N/A

### Notify maintainers

@bgamari @jansol @Mic92

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-linux"`
 - host os: `Linux 5.15.12, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/libraries/audio/roc-toolkit/default.nix
+++ b/pkgs/development/libraries/audio/roc-toolkit/default.nix
@@ -49,6 +49,9 @@ stdenv.mkDerivation rec {
        else [ "--with-libraries=${openfec}/lib"
               "--with-openfec-includes=${openfec.dev}/include" ]);
 
+  prePatch = lib.optionalString stdenv.isAarch64
+    "sed -i 's/c++98/c++11/g' SConstruct";
+
   preConfigure = ''
     sconsFlags+=" --prefix=$out"
   '';

